egs/timit/ASR/shared/parse_options.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
This file is shared across various recipes.

Could you make it a symlink, like what we are doing in the librispeech recipe?